xAPI-Communication.md

```diff
@@ -324,26 +318,25 @@ of Statements which contain only attachment Objects with a populated fileUrl.
 __Note:__ There is no requirement that Statement batches using the mime/multipart format
 contain attachments.
 
-###### Client Requirements
+###### Learning Record Producer Requirements
```

Comment:
Sigh, and I was so sure I didn't do it that I didn't do a sweep.  Now I did a sweep.